metricbeat/docs/fields.asciidoc

```diff
@@ -1687,15 +1687,15 @@ Number of requested checkpoints that have been performed.
 [float]
 === postgresql.bgwriter.checkpoints.times.write.ms
 
-type: float
+type: scaled_float
```

Comment:
I'd leave these as floats because they are the total (not the derivative) number of milliseconds, so if you have an instance running for a year these numbers can be huge. Since float is what Postgres uses to report them, I'd say it's safer for us to also use floats to avoid overflows.